lib/plugins/package/lib/packageService.js

```diff
@@ -4,6 +4,7 @@ const BbPromise = require('bluebird');
 const path = require('path');
 const globby = require('globby');
 const _ = require('lodash');
+const nanomatch = require('nanomatch');
```

Comment:
Wow. @micromatch publishes `micromatch`, `nanomatch` `picomatch`, and `anymatch`. That's not confusing at all 🙄

Reply:
We had our fun with glob and *match yesterday :) Globby uses fast glob. Fast glob only uses default wildcard pattern according to their docs. However internally they use micromatch, not nanomatch. So they might actually expose the extended syntax that it provides over nanomatch.

We could start using micromatch instead of nanomatch here. I have to check what fast glob does internally. Maybe that's a test we could be adding. This whole thing is so confusing haha... Would have loved to fix it downstream, but honestly don't have the time right now and this is pressing.

Somewhat unrelated but interesting:
- Globby seems so calls fast glob for each pattern which results in many file scans. Who wrote that and thought it was performant?! All we did is replicate that functionality in memory. 
- Still not entirely sure what the difference between picomatch and nanomatch really is. No dependencies for picomatch?
- We also have npm glob which provides its own custom syntax and uses minimatch (which is apparently slow).